studio/next.config.mjs

```diff
@@ -1,11 +1,30 @@
 import withMarkdoc from "@markdoc/next.js";
+import { withSentryConfig } from "@sentry/nextjs";
 import pkg from "./package.json" assert { type: "json" };
 
 const isPreview = process.env.VERCEL_ENV === "preview";
 // Allow it only for development once https://github.com/vercel/next.js/issues/23587 is fixed
 const allowUnsafeEval = true;
 // Report CSP violations to the console instead of blocking them
 const debugCSP = false;
```

Comment:
Do we need to adjust our CSP policies to allow sentry as trusted origin? You can test this by running the production build.

Reply:
Checked this, it needed at least `blob` for the web worker.
`connect-src` can't be enforced atm.

Reply:
ok